nf_core/lint/nextflow_config.py

```diff
@@ -262,6 +268,48 @@ def nextflow_config(self):
                 )
             )
 
+    # Check if custom profile params are set correctly
+    if self.nf_config.get("params.custom_config_version", "").strip("'") == "master":
+        passed.append("Config `params.custom_config_version` is set to `master`")
+    else:
+        failed.append("Config `params.custom_config_version` is not set to `master`")
+
+    custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/" + self.nf_config.get(
+        "params.custom_config_version", ""
+    ).strip("'")
+    if self.nf_config.get("params.custom_config_base", "").strip("'") == custom_config_base:
+        passed.append("Config `params.custom_config_base` is set to `{}`".format(custom_config_base))
+    else:
+        failed.append("Config `params.custom_config_base` is not set to `{}`".format(custom_config_base))
+
+    # Check that lines for loading custom profiles exist
+    lines = [
+        r"// Load nf-core custom profiles from different Institutions",
+        r"try {",
+        r'includeConfig "${params.custom_config_base}/nfcore_custom.config"',
+        r"} catch (Exception e) {",
+        r'System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")',
+        r"}",
+    ]
+    path = os.path.join(self.wf_path, "nextflow.config")
+    i = 0
+    with open(path, "r") as f:
+        for line in f:
+            if lines[i] in line:
+                i += 1
+                if i == len(lines):
+                    break
+            else:
+                i = 0
+    if i == len(lines):
+        passed.append("Lines for loading custom profiles found")
+    else:
+        lines[2] = "\t" + lines[2]
+        lines[4] = "\t" + lines[4]
+        failed.append(
+            "Lines for loading custom profiles not found. File should contain: ```groovy\n{}".format("\n".join(lines))
```

Comment:
This still prints the entire block - I was more thinking of printing the exact line(s) that were missing. The reason being that this kind of test can often fail because of an extra space here or there, or double quote instead of single etc. So sometimes it'll be the whole block that's not there but sometimes it'll just be a single line. In which case it can be nice to know which line is wrong.

Minor though..

Reply:
Let's forget about this for now 👍🏻